### PR TITLE
Group details

### PR DESCRIPTION
**Pull request by corsacca**

### CODE REVIEW

File: dt-groups/groups-post-type.php [hunk 1]
```diff
@@ -686,23 +689,6 @@ public function get_custom_fields_settings () {
                     'section' => 'address'
                 ];
             }
-        } else {
```
Comment (ChrisChasm) [inline dt-groups/groups-post-type.php]: Are you removing the ability to have specific addresses or multiple addresses attached to groups? I have an interest in keeping this. I think specifically for the way I intend to use it with Zume and tracking location of individual groups. Locations can give a region and be used for assigning coaches and tracking area activity, but specifically addresses for groups can produce marker based maps. This is currently how I'm producing maps for the Zume group. 

Am I off? This seems like what is being removed here.

Comment (corsacca) [reply dt-groups/groups-post-type.php]: @ChrisChasm, they functionality is still there. It just won't create a "default" address.

Comment (corsacca) [reply dt-groups/groups-post-type.php]: Each contact and each group can have 0 or as many addresses as needed. 
They address have a key (address_111) and a value ("123 example road...)
Each address has an address details key (address_111_details) ([primary:true, verified:false])
Those fields are not in the database until the address in created

Comment (ChrisChasm) [reply dt-groups/groups-post-type.php]: Oh ya. I remember that conversation from a while ago about the address. Great. I'm tracking.

File: dt-groups/groups.php [hunk 2]
```diff
@@ -237,9 +237,18 @@ public static function add_location_to_group( $group_id, $location_id ){
             array('date' => current_time( 'mysql' ) )
         );
     }
+    public static function add_member_to_group( $group_id, $member_id ){
```
Comment (burntsun) [inline dt-groups/groups.php]: Great method! Good name! (Maybe just add types?)

**Pull request merged**